docs/guides/user-containers/readme.md

```diff
@@ -99,11 +147,11 @@ Hub username and the email address they have associated with that username.  In
 this case, the secret will be named `readonly-red-rock-slushy`.
 
 ```console
-$ USER_TOKEN=users-token-text
-$ USER_NAME=dean
-$ USER_EMAIL=dean@myco.com
-$ SECRET_NAME=readonly-red-rock-slushy
-$ kubectl create secret docker-registry $SECRET_NAME -n default --docker-server="https://index.docker.io/v1/" --docker-username=$USER_NAME --docker-password=$USER_TOKEN --docker-email=$USER_EMAIL
+USER_TOKEN=users-token-text
```

Comment:
I tend to get disoriented without the prompt, but I guess this is a regular area where heads bump.

Reply:
I use them only when there is output. My markdown linter auto-corrected it: https://github.com/DavidAnson/markdownlint/blob/v0.29.0/doc/md014.md.